app/src/main/res/layout/component_settings_theme_option.xml

```diff
@@ -20,6 +21,6 @@
     android:layout_height="wrap_content"
     android:src="@drawable/bpk_tick"
     android:layout_weight=".2"
-    android:tint="?attr/colorPrimary"
+    app:tint="?attr/colorPrimary"
```

Comment:
Lint complains about this. New approach is the official way it needs to be written. It's also more compatible.